tests/unit/test_minimax_tts_worker.py

```
@@ -164,26 +183,72 @@ async def task_on_send(ws, message):
+
+    request_queue = ControlledQueue()
+    response_queue = queue.Queue()
+    thread = _start_worker(request_queue, response_queue)
+
+    _wait_for_queue_item(response_queue, lambda item: item == ("__ready__", True))
+
+    request_queue.put(("speech-1", "你好"))
+    request_queue.put(("speech-1", "世界今天"))
+    request_queue.put((None, None))
+
+    audio_item, seen = _wait_for_queue_item(
```

Ruff (0.15.10)

[warning] Unpacked variable `seen` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)